scripts/test_quickstart.sh

```diff
@@ -16,11 +16,12 @@
 # Run a CI `script` phase to build the associated quickstart
 # sample and run its tests.
 
-if [[ "$TRAVIS_PULL_REQUEST" == "false" ||
-      "$TRAVIS_PULL_REQUEST_SLUG" == "$TRAVIS_REPO_SLUG" ]]; then
+set -xeuo pipefail
+
+# Set have_secrets to true or false.
+. scripts/check_secrets.sh
```

Comment:
When sourcing scripts, consider spelling out `source` rather than `.`. It makes this more obvious to the reader that you're not just calling the script but including into the currently running script.

Reply:
Done